src/containers/sidebar/index.js

```diff
@@ -3,11 +3,60 @@ import { Route } from 'react-router-dom'
 import Login from '../login'
 import Register from '../register'
 import Body from '../body'
+import Avatar from 'material-ui/Avatar';
+import List from 'material-ui/List';
+import TextField from 'material-ui/TextField';
+import ListItem from 'material-ui/List/ListItem';
+import MuiThemeProvider from 'material-ui/styles/MuiThemeProvider';
+import SearchBar from 'material-ui-search-bar'
+import Chip from 'material-ui/Chip';
+import Avi from "./avi.jpg"
+
+import './index.css'
+import 'typeface-roboto'
+
+const styles = {
+    chip: {
+      margin: 4,
+      width: 300,
+    },
+    wrapper: {
+      display: 'flex',
+      flexWrap: 'wrap',
+    },
+  };
+
+const style = {margin: 5};
 
 const Sidebar = () => (

```

Comment:
I don't think the () => part is needed. I think it can just be const Sidebar = (<...>). Because now you're creating a constant function.

Reply:
When you use it then you can import it like you would normally then render() {return Sidebar}